src/backend/columnar/columnar_tableam.c

```diff
@@ -2984,3 +3055,23 @@ InstalledExtensionVersionColumnar(void)
 
 	return installedExtensionVersion;
 }
+
+
+/*
+ * GetExtensionOption returns DefElem * node with "defname" from "options" list
+ */
+DefElem *
+GetExtensionOption(List *extensionOptions, const char *defname)
+{
+	DefElem *defElement = NULL;
+	foreach_ptr(defElement, extensionOptions)
+	{
```

Comment:
I'm a bit surprised none of this is showing up in code coverage. Is it not covered, or some issue with the code coverage tool?